app/Http/Controllers/Admin/BookController.php

```diff
@@ -34,4 +33,22 @@ public function index()
         $books  = Book::with('borrowings')->withCount('borrowings')->sortable()->paginate(config('define.page_length'));
         return view('backend.books.list', compact('books'));
     }
+
+    /**
+     * Show the form with book data for edit book.
+     *
+     * @param int $id book
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
```

Comment:
u should get object param edit(Book $book)

Reply:
không cần thiết

Reply:
u set param id and then u call Book object where id = $id???? why don't u set param = object???